### ISSUE TITLE
Provide a way to share a snippet

### ISSUE THREAD

**Issue by Totktonada** [tarantool/try#30]
Like on https://godbolt.org or https://jsfiddle.net

**Comment by tsafin** [comment]
Yes, yes, please do it - we do need something to share code snippets without huge pasta to Telegram chat.
